app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    instructions = command.split()
+    _, source, destination = instructions
+
+    if len(instructions) != 3 or instructions[0] != "mv":
+        raise ValueError("Incorrect command")
+
+    if os.path.exists(destination):
+        raise FileExistsError("Already exists!")
+
+    path = os.path.join("/".join(destination.split("/")[:-1]))
```

Comment:
The usage of `os.path.join` here is incorrect. You should pass each part of the path as separate arguments to `os.path.join` instead of joining them with `/` manually. This ensures cross-platform compatibility as per the checklist requirement: 'Use `os.path.join` to concatenate parts of the path.'